dozer-types/src/helper.rs

```diff
@@ -75,6 +97,19 @@ impl Field {
                     })
                 }
             }
+            FieldType::U128 => {
```

Comment:
I don't think we should convert `"null"` to `Field::Null`

Reply:
other types are all supporting in that way, I just followed the way we defined `UInt` and `Int`

Reply:
Can you double check? Last time I checked it converting `"null"` to `Field::UInt` results in an `Err`.

Reply:
https://github.com/getdozer/dozer/pull/1315/files/eafbddaef0509856831ef4022611164e3d4749b8#diff-6f3491eceadec0a62a4343f7e179d5b6eea02801eaa4efeb32e37880b2662f87R88

Reply:
Great thanks!